neqo-transport/src/connection/tests/recovery.rs

```diff
@@ -255,38 +256,47 @@ fn pto_handshake_complete() {
     assert_eq!(*client.state(), State::Connected);
 
     let cb = client.process_output(now).callback();
-    assert_eq!(cb, HALF_RTT * 6);
+    assert_eq!(cb, pto);
 
     let mut pto_counts = [0; MAX_PTO_COUNTS];
     assert_eq!(client.stats.borrow().pto_counts, pto_counts);
 
     // Wait for PTO to expire and resend a handshake packet.
     // Wait long enough that the 1-RTT PTO also fires.
     qdebug!("---- client: PTO");
-    now += HALF_RTT * 6;
+    now += pto;
     let pkt2 = client.process_output(now).dgram();
     assert_handshake(pkt2.as_ref().unwrap());
 
     pto_counts[0] = 1;
     assert_eq!(client.stats.borrow().pto_counts, pto_counts);
 
+    // PTO has been doubled.
+    let pto = 2 * pto;
+    let cb = client.process_output(now).callback();
+    assert_eq!(cb, pto);
```

Comment:
After a PTO expiry, the PTO doubles. Previously the test did not account for that, and thus trigger a time threshold based loss detection, but not a second PTO.

With the fix of this pull request, and the above accounting for PTO doubling, the test now actually sees a second PTO as expected.